samples/add-raster-from-service/src/main/java/com/esri/arcgismaps/sample/addrasterfromservice/screens/AddRasterFromServiceScreen.kt

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.addrasterfromservice.screens
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.LoadStatus
+import com.arcgismaps.toolkit.geoviewcompose.MapView
+import com.esri.arcgismaps.sample.addrasterfromservice.components.AddRasterFromServiceViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.LoadingDialog
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app.
+ * Displays a MapView showing a raster retrieved from an image service.
+ */
+@Composable
+fun AddRasterFromServiceScreen(sampleName: String) {
+    val mapViewModel: AddRasterFromServiceViewModel = viewModel()
+
+    // Observe raster load status to show a simple loading overlay while the raster is being fetched/drawn
+    val rasterLoadStatus by mapViewModel.rasterLoadStatus.collectAsStateWithLifecycle(initialValue = LoadStatus.NotLoaded)
```

Comment:
The initial value can be derived from the viewmodel: 
```suggestion
    val rasterLoadStatus by mapViewModel.rasterLoadStatus.collectAsStateWithLifecycle()
```